helmfile/charts/prometheus-alerts/values.yaml

```diff
@@ -118,6 +118,7 @@ defaultRules:
     dns: true
     clusterApi: false
     harbor: true
+    diskPerf: true
```

Comment:
if we want this enabled by default you should disable it for wc, as only the admins should see and act on these
adding `defaultRules.rules.diskPerf: true` [here](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/prometheus-user-alerts-wc.yaml.gotmpl) should be enough

Reply:
@crssnd not `false` if it should be disabled for wc?

Reply:
Would this be more fitting under `.diskAlerts` ?